<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuctionWindow</class>
 <widget class="QDialog" name="AuctionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="cardWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlaceButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Place bid</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="PlayerBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>101</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGraphicsView" name="PlayerIconView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>160</green>
          <blue>160</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="BiddingSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="BiddingSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>20</width>
      <height>311</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't bid</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BiddingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BiddingSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BiddingSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BiddingSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
